website/docs/Examples/Tune-PyTorch.md

```diff
@@ -283,4 +284,4 @@ Files already downloaded and verified
 Best trial test set accuracy: 0.6294
 ```
 
-[Link to notebook](https://github.com/microsoft/FLAML/blob/main/notebook/tune_pytorch.ipynb) | [Open in colab](https://colab.research.google.com/github/microsoft/FLAML/blob/main/notebook/tune_pytorch.ipynb)
\ No newline at end of file
+[Link to notebook](https://github.com/microsoft/FLAML/blob/main/notebook/tune_pytorch.ipynb) | [Open in colab](https://colab.research.google.com/github/microsoft/FLAML/blob/main/notebook/tune_pytorch.ipynb)
```

Comment:
Update the notebook too.
Also, check whether [test_pytorch_cifar10.py](https://github.com/microsoft/FLAML/blob/main/test/tune/test_pytorch_cifar10.py) needs to be updated.

Reply:
I've updated the notebook and the test